tools/memap.py

```diff
@@ -389,67 +409,56 @@ def generate_output(self, export_format, file_output=None):
                 k:self.modules[i][k] for k in self.print_sections}})
 
             table.add_row(row)
+        json_obj.append({\
+                'summary':{\
+                'static_ram':(subtotal['.data']+subtotal['.bss']),\
+                'heap':(subtotal['.heap']),\
+                'stack':(subtotal['.stack']),\
+                'total_ram':(subtotal['.data']+subtotal['.bss']+subtotal['.heap']+subtotal['.stack']),\
+                'total_flash':(subtotal['.text']+subtotal['.data']+misc_flash_mem),}})
+
+        file_desc.write(json.dumps(json_obj, indent=4))
+        file_desc.write('\n')
+
+    def export_csv_ci(self, file_desc, _, subtotal, misc_flash_mem):
+        csv_writer = csv.writer(file_desc, delimiter=',', quoting=csv.QUOTE_NONE)
+
+        csv_module_section = []
+        csv_sizes = []
+        for i in sorted(self.modules):
+            for k in self.print_sections:
+                csv_module_section += [i+k]
+                csv_sizes += [self.modules[i][k]]
 
-        subtotal_row = ['Subtotals']
-        for k in self.print_sections:
-            subtotal_row.append(subtotal[k])
-
-        table.add_row(subtotal_row)
-
-        if export_format == 'json':
-            json_obj.append({\
-                  'summary':{\
-                  'static_ram':(subtotal['.data']+subtotal['.bss']),\
-                  'heap':(subtotal['.heap']),\
-                  'stack':(subtotal['.stack']),\
-                  'total_ram':(subtotal['.data']+subtotal['.bss']+subtotal['.heap']+subtotal['.stack']),\
-                  'total_flash':(subtotal['.text']+subtotal['.data']+misc_flash_mem),}})
-
-            file_desc.write(json.dumps(json_obj, indent=4))
-            file_desc.write('\n')
-
-        elif export_format == 'csv-ci': # CSV format for the CI system
-
-            csv_writer = csv.writer(file_desc, delimiter=',', quoting=csv.QUOTE_NONE)
-
-            csv_module_section = []
-            csv_sizes = []
-            for i in sorted(self.modules):
-                for k in self.print_sections:
-                    csv_module_section += [i+k]
-                    csv_sizes += [self.modules[i][k]]
```

Comment:
I just tried doing some builds from master and from this PR and it looks like the output was affected.

## Master
```
Building project detect (RZ_A1H, ARM)
Compile: main.cpp
Compile: test_env.cpp
Link: detect
Elf2Bin: detect
+-----------+-------+-------+--------+
| Module    | .text | .data |  .bss  |
+-----------+-------+-------+--------+
| Misc      | 40479 |  246  | 626804 |
| Subtotals | 40479 |  246  | 626804 |
+-----------+-------+-------+--------+
Static RAM memory (data + bss): 627050
Heap: 0
Stack: 0
Total RAM memory (data + bss + heap + stack): 627050
Total Flash memory (text + data + misc): 40725
```

## PR
```
Building project detect (RZ_A1H, ARM)
Compile: main.cpp
Compile: test_env.cpp
Link: detect
Elf2Bin: detect
+-----------+-------+-------+------+
| Module    | .text | .data | .bss |
+-----------+-------+-------+------+
| Subtotals |   0   |   0   |  0   |
+-----------+-------+-------+------+
Static RAM memory (data + bss): 0
Heap: 0
Stack: 0
Total RAM memory (data + bss + heap + stack): 0
Total Flash memory (text + data + misc): 0
```

So even if it shouldn't change anything, it looks like it is. So perhaps this change should be in another PR?